### ISSUE TITLE
t490 boot problem - AAPL: [EB|#LOG:EXITBS:START]

### ISSUE THREAD

**Issue by gccalex** [5T33Z0/Thinkpad-T490-Hackintosh-OpenCore#29]
Thank you very much for taking so much time and effort to upload thorough and detailed instructions to install hacintosh on t490, I really do appreciate that!!!
I do have Lenovo ThinkPad T490 (20N2) model.
CPU: Intel Core i5 8365U
iGPU: Intel(R) UHD Graphics 620
WIFI: Intel AC-9560
 latest UEFI and BIOS Firmware
I followed your guide thoroughly and prepared USB drive with your latest EFI release + Monterey Recovery (it has been done in Windows) + generated a new SMBIOS MacBookPro15,2
BIOS setting changed according to the instruction you have provided. 
I do have a difficulty booting from USB…. Loading is stuck at  AAPL: [EB|#LOG:EXITBS:START]
I haven’t changed much in the config file, other than tried other framebuffer patches you have provided (including additional list of Framebuffer Patches from your repository). All the kexts, drivers and ACPI files are in it’s place. (Just as you have it in your EFI release folder)
I also set Misc -> Debug -> Target -> 67 (for debug prepossess, to get the logs print out to the EFI folder)
 What has been done so far: I did followed recommendations from dortania website on troubleshooting Kernel Issues trying different combination of booting and UEFI quirks. Nothing helped so far… still stuck at AAPL: [EB|#LOG:EXITBS:START]
Would you be so kind to help me out with my config file, maybe I can drop you my config file or EFI folder. 
Thank you.

**Comment by 5T33Z0** [comment]
My latest EFI has only been tested with macOS Sonoma and Sequoia. So I don't know if it's stll working with Monterey. I suggest, you try one of the older releases in this case.

Since I have a real mac to create a full USB installer I've never used the Online Recovery method to download and install macOS. You either have to attach a LAN cable to download the OS, since `itlwm.kext` doesn't work in Recovery, or you need to use the Monterey Variant of `AirportItlwm.kext` (and disable itlwm.kext) to use WiFi during recovery/installation.

**Comment by gccalex** [comment]
Thank you very much for your reply and advice!
I was able to successfully install MacOS to my t490!
I have used the real Mac to fully download MacOS and used your EFI folder with 0.9.7 OpenCore (as I needed Monterey) 
I also used nvme=-1 boot arg. To turn of my NVMe drive and immediately booted into MacOS installer. I installed MacOS into external drive. Seems like I need compatible NVMe for my T490. 
Thank you very much for your help!!!!!
